src/Lean/Environment.lean

```diff
@@ -765,7 +765,8 @@ partial def importModulesCore (imports : Array Import) : ImportStateM Unit := do
       moduleNames := s.moduleNames.push i.module
     }
 
-def finalizeImport (s : ImportState) (imports : Array Import) (opts : Options) (trustLevel : UInt32 := 0) : IO Environment := do
+def finalizeImport (s : ImportState) (imports : Array Import) (opts : Options) (trustLevel : UInt32 := 0)
```

Comment:
Please add documentation for `leakEnv`

Reply:
And motivate why this was added. We will forget the discussion we had this week.

Reply:
Yes, I wasn't sure where to document it. I've split the comment below, moved the first part into the docstring, and extended comments in general.